src/XRP/model/xrp-issued-currency.ts

```diff
@@ -27,6 +35,12 @@ export default class XRPIssuedCurrency {
     return undefined
   }
 
+  /**
+   *
```

Comment:
No description for this comment?

Reply:
Nope, you'll notice that none of the private constructors have any comments apart from documenting the individual fields.  These classes expose a different constructor which wraps the private constructor, and that's the method that gets fully documented.